genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings.go

```diff
@@ -47,7 +41,24 @@ func createPhoneBaseSettings(ctx context.Context, d *schema.ResourceData, meta i
 	sdkConfig := meta.(*provider.ProviderMeta).ClientConfig
 	phoneBaseProxy := getPhoneBaseProxy(sdkConfig)
 
-	log.Printf("Creating phone base settings %s", name)
+	log.Printf("Getting phone base settings template for %s", phoneMetaBase)
+	phoneBaseSettingTemplate, resp, err := phoneBaseProxy.getPhoneBaseSettingTemplate(ctx, *phoneMetaBase.Id)
+	if err != nil {
+		return util.BuildAPIDiagnosticError(resourceName, fmt.Sprintf("Failed to get phone base settings template %s error: %s", phoneMetaBase, err), resp)
+	}
+
+	phoneBaseSettingTemplateLines := *phoneBaseSettingTemplate.Lines
+	if len(phoneBaseSettingTemplateLines) == 0 {
+		return util.BuildAPIDiagnosticError(resourceName, fmt.Sprintf("Failed to get phone base settings template lines for %s", phoneMetaBase), resp)
+	}
+	phoneBase.Lines = &[]platformclientv2.Linebase{
+		{
+			Name:         &name,
+			LineMetaBase: phoneBaseSettingTemplateLines[0].LineMetaBase,
```

Comment:
can we have more than 1 phoneBaseSettingTemplateLines ? and are we sure it is always the 1st one from the api that need to be assigned for linemetabase ?

Reply:
the template always only returns a single instance of the line meta base.